pkg/plugins/runtime/k8s/controllers/gatewayapi/http_route_controller.go

```diff
@@ -105,148 +112,165 @@ func (r *HTTPRouteReconciler) Reconcile(ctx context.Context, req kube_ctrl.Reque
 
 type ParentConditions map[gatewayapi.ParentReference][]kube_meta.Condition
 
+func generateMeshHTTPRouteName(route *gatewayapi.HTTPRoute, parentRef gatewayapi.ParentReference) string {
+	hash := fnv.New32()
+
+	parentRefElems := slices.DeleteFunc(
```

Comment:
I think these can just go in the hash without filtering them right?

Reply:
You are right, this is leftover when `parentRefElems` were used to generate name. Removed filtering logic